Shared/AuthenticationKeychainInfo.swift

```diff
@@ -25,7 +25,7 @@ public extension KeychainWrapper {
     }
 }
 
-open class AuthenticationKeychainInfo: NSObject, NSCoding {
+open class AuthenticationKeychainInfo: NSObject, NSCoding, NSSecureCoding {
```

Comment:
You don't have to have both `NSCoding` and `NSSecureCoding`

Reply:
yeah didn't notice, Thanks